src/vmm/src/rpc_interface.rs

```diff
@@ -555,7 +560,10 @@ impl<'a> PrebootApiController<'a> {
 
         let elapsed_time_us =
             update_metric_with_elapsed_time(&METRICS.latencies_us.vmm_load_snapshot, load_start_us);
-        info!("'load snapshot' VMM action took {} us.", elapsed_time_us);
+        info!(
```

Comment:
All other mesasages are warning while this is info. Should we make this aw arning as well for consistency ?

Reply:
The log line's purpose is a bit overloaded conveying dev-preview as well as latency. I will change the log level to warning given that is the more "urgent" information coming from the log.